test/inductor/test_aot_inductor.py

```diff
@@ -3785,6 +3811,7 @@ def fail_cuda(is_skip=False):
     # quantized unsupported for GPU
     "test_quantized_linear": fail_cuda(is_skip=True),
     "test_quanatized_int8_linear": fail_cuda(is_skip=True),
+    "test_tensor_constant": fail_cuda(is_skip=True),
```

Comment:
Why does it fail for cuda?

Reply:
I would need to designate the device for creating the tensor. We can definitely pass in with different devices, but this is mainly used to make sure that with symbolic_trace we do have TensorConstant type being used.